src/ui/components/filters/filteroptionscomponent.js

```diff
@@ -634,9 +646,6 @@ export default class FilterOptionsComponent extends Component {
       default:
         throw new AnswersComponentError(`Unknown optionType ${this.config.optionType}`, 'FilterOptions');
     }
-    if (!this.config.isDynamic) {
-      this.saveSelectedWithPersist();
```

Comment:
Confirming: this is moved to core now, right? Would someone still see filters in their URL if they did not have a filter options component? I don't think this is a case we need to worry about, just bringing it up. It seems like if someone were adding their filters without the component, they should be responsible for, during any upgrade, to make sure it's not in the URL after a search.

Reply:
ah, yeah even if they don't have any filter components they will see `filters={}` in the url. With the current implementation he param won't do anything without corrresponding static filter components, though.